tests/common/plugins/conditional_mark/__init__.py

```diff
@@ -126,30 +140,31 @@ def load_basic_facts(session):
 
     return results
 
-
-def find_longest_match(nodeid, case_names):
+def find_longest_match(nodeid, conditions):
```

Comment:
Since we may have multiple matches now. Suggest to rename the function to `find_longest_matches`.

Reply:
done